### PR TITLE
Case 21618: menithal's avatar is constantly flashing

### PR DESCRIPTION
**Pull request by danteruiz**
ticket - https://highfidelity.manuscript.com/f/cases/21618/menithal-s-avatar-is-constantly-flashing

### CODE REVIEW

File: libraries/entities-renderer/src/RenderableModelEntityItem.cpp [hunk 1]
```diff
@@ -181,9 +181,11 @@ void RenderableModelEntityItem::updateModelBounds() {
         updateRenderItems = true;
     }
 
-    if (model->getScaleToFitDimensions() != getScaledDimensions() ||
-            model->getRegistrationPoint() != getRegistrationPoint() ||
-            !model->getIsScaledToFit()) {
+    bool overridingModelTransform = model->isOverridingModelTransformAndOffset();
+        if (!overridingModelTransform &&
```
Comment (samcake) [inline libraries/entities-renderer/src/RenderableModelEntityItem.cpp]: just the tab :)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18046/HighFidelity-Beta-PR15153-4371723.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/18046/HighFidelity-Quest-Beta-PR15153-4371723.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18046/HighFidelity-Beta-PR15153-4371723.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/18046/HighFidelity-Beta-Interface-PR15153-4371723.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18046/HighFidelity-Beta-PR15153-4371723.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/18046/HighFidelity-Beta-Interface-PR15153-4371723.exe

**Pull request merged**